### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by Bonymol-aot**
# Issue Tracking

JIRA: 
Issue Type: BUG/ FEATURE

# Changes

Changelog updated with the change on FORMIO_JWT_SECRET.

# How has the change been tested?

# Screenshots (if applicable)

# Build Success screenshot (Till a CICD pipeline is set up)

# Notes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -112,6 +112,7 @@ Mark  items as `Added`, `Changed`, `Fixed`, `Removed`, `Untested Features`, `Upc
 
 * Removed View submissions button from reviewer form list and view submissions route.
 * Removed the environment variables `CLIENT_ROLE_ID`, `DESIGNER_ROLE_ID`, `REVIEWER_ROLE_ID`,`ANONYMOUS_ID`, `USER_RESOURCE_ID`.
+* Removed the environment variable `FORMIO_JWT_SECRET`.
```
Comment (abhilash-aot) [inline CHANGELOG.md:115]: Add removal at config.json react_app_jwt secret

**Review by abhilash-aot — COMMENTED** [review]
Missing Changelog Info in Env updated

**Review by abhilash-aot — APPROVED** [review]

**Review by abilpraju-aot — APPROVED** [review]
LGTM

**Review by sreehari-aot — APPROVED** [review]
LGTM

**Pull request merged**